proposed/container.md

```diff
@@ -34,6 +34,8 @@ Users of dependency injections containers (DIC) are referred to as `user`.
 
 - `has` takes one unique parameter: an entry identifier. It MUST return `true`
   if an entry identifier is known to the container and `false` if it is not.
+  `has($id)` returning true does not mean that `get($id)` will not throw an exception.
+  It does however mean that `get($id)` will not throw a `NotFoundException`.
```

Comment:
:+1: for this part as this is backporting this PR to the standard: https://github.com/container-interop/container-interop/pull/45